libs/dh/dev-examples/feature-processes/src/details.component.ts

```diff
@@ -80,6 +84,11 @@ import { DhCalculationsDetailsGridAreasComponent } from './gridareas.component';
           </dh-process-state-badge>
         }
       </watt-drawer-topbar>
+      <watt-drawer-actions>
+        <watt-button variant="secondary" (click)="navigation.navigate('edit', process?.id)">{{
+          t('edit.editButtonTitle')
+        }}</watt-button>
```

Comment:
nitpick: just personal preference when whitespace is not of importance:

```suggestion
        <watt-button variant="secondary" (click)="navigation.navigate('edit', process?.id)">
          {{ t('edit.editButtonTitle') }}
        </watt-button>
```